src/instrument.js

```diff
@@ -70,7 +70,7 @@ function computeNextEntry(reducer, action, state, error) {
   let nextState = state;
   let nextError;
   try {
-    nextState = reducer(state, action);
+    nextState = reducer(state, { ...action, replaying });
```

Comment:
@gaearon Only issue I am seeing here is that `IsReplaying` Symbol would have to be exposed by `redux-devtools`. However, that would mean that any library which relies on the flag, would need to define `redux-devtools` as `peerDependency`.

Not sure if that's acceptable.